src/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -207,6 +207,11 @@ See the LICENSE file in the project root for more information.
       <IlcArg Condition="$(IlcGenerateCompleteTypeMetadata) == 'true'" Include="--completetypemetadata" />
       <IlcArg Condition="$(IlcGenerateStackTraceData) == 'true'" Include="--stacktracedata" />
       <IlcArg Condition="$(RootAllApplicationAssemblies) == 'true'" Include="--rootallapplicationassemblies" />
+      <IlcArg Include="--directpinvoke:^\*$" />
+      <IlcArg Include="--directpinvoke:^\[MRT\]$" />
+      <IlcArg Include="--directpinvoke:^__Internal$" Condition="'$(IlcDirectPInvokeInternal)' != 'false'" />
+      <IlcArg Include="--directpinvoke:^System[.].*" Condition="'$(TargetOS)' != 'Windows_NT'" />
+      <IlcArg Include="--directpinvoke:^api-ms-win-.*" Condition="'$(TargetOS)' == 'Windows_NT'" />
```

Comment:
We are going to hit a problem with `api-ms-win-*` sooner or later. The `api-ms-win-*` APIs are often used for light up on new Windows versions and folks do want to use lazy lookup for them to make the light up work. I think we may need more fine grained way to specify this  - e.g. per assembly or something.

Reply:
Or be able to say "All entrypoints in this one; except ....`.

Reply:
Per assembly might be overkill. I structured it so that within the compiler we have this at `module:symbol` granularity. Can you think of a scenario where that level of control wouldn't be sufficient?

I didn't go as far as exposing it from the command line because the idea of parsing the regex to split it into a module+symbol felt like too much work to get right and test.

Reply:
`module:symbol` is sufficient to make things work, but it may not be easy to get it right if you have apps with light-up. There are conflicting requirements:

- We do require a subset of PInvokes used by CoreLib to be direct to make things work
- We would prefer PInvokes that are available everywhere to be direct
- Apps with lightup require subset of PInvokes to be lazy to make things work

Maybe we should allow specifying a text file(s?) with a list direct PInvokes on the command line. We would include a text file with Win32 APIs that are available everywhere (or just the common subset - kernel32 and friends) with the compiler and pass it on the command line by default.

If we have done that, we can probably abandon the regex idea. Basically, we would have two controls for this:
- The coarse grained control on the command line that is per-library. No need to have regexes. 
- The fine grained control in the files that let you specify the exact lists of what to PInvoke directly.